src/api/rest/v1/ds/routes.ts

```diff
@@ -5,8 +5,10 @@ const router = express.Router()
 
 router.get(/get\/(.*)\/(.*)$/, controller.getById)
 router.post(/query\/(.*)$/, controller.query)
-router.delete(/([^\/]*)$/, controller.delete)
+router.delete("/:schema", controller.delete)
```

Comment:
For RESTful, the record ID should be as param as well `/:schema/:recordId` rather than in an `id` query param.
And I'd do two endpoints for the whole datastore `DELETE /:schema` or an individual record `DELETE /:schema/:recordId` rather than the `destroy`/`delete`
`DELETE /:schema` is risky but the same level of risk exists by confusing `destroy` and `delete` even with the record `id`

Reply:
I guess the query `id` is used in the web UI, not a big deal for this PR

Reply:
I've added support for `DELETE /:schema/:recordId` and retained support for `DELETE /:schema` with `recordId` in query params for backwards compatibility so you can update the UI.

I prefer to use `destroy` as a query param to avoid accidential confusion as you have to explicitly add that param rather than make a generic DELETE request.

Reply:
> For RESTful, the record ID should be as param as well `/:schema/:recordId` rather than in an `id` query param.

Done